Automattic/jetpack
# Instant Search: Add Jetpack Search gridicon

<img width="700" alt="Screen Shot 2020-02-10 at 4 15 55 PM" src="https://user-images.githubusercontent.com/4044428/74199166-a2aafa00-4c20-11ea-8b7b-aedfa191e8a3.png">

Fixes #12540.

#### Changes proposed in this Pull Request:
* Adds [icon A](https://github.com/Automattic/jetpack/issues/12540#issuecomment-578055331) as Jetpack Search gridicon.
* Place Jetpack Search gridicon next to overlay search input.
* Adjust overlay search input styling.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:
1. Follow [these setup instructions](https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md#testing-instructions) to set up Jetpack Instant Search on your site. Please note that the Jetpack Search widget will need to be added to twice: once to your site sidebar/footer and once to the Jetpack Search Sidebar within the search overlay.
2. Search your site to trigger the Jetpack Search overlay.
3. Ensure that the Jetpack Search gridicon is rendered adjacent to the overlay search input.

#### Proposed changelog entry for your changes:
* None.
